src/components/Post/Comment/EditableReplyContent.tsx

```diff
@@ -240,53 +240,62 @@ const EditableReplyContent = ({ userId, className, commentId, content, replyId ,
 		}
 	};
 
-	const deleteReply = async () => {
-		setLoading(true);
-		const { data, error: deleteReplyError } = await nextApiClientFetch<MessageType>('api/v1/auth/actions/deleteCommentReply', {
-			commentId,
-			postId: ((reply.post_index || reply.post_index === 0)? reply.post_index: postIndex),
-			postType: reply.post_type || postType,
-			replyId,
-			trackNumber: track_number
+	const removeReplyContent = async() => {
+		const keys = Object.keys(comments);
+		setComments((prev:any) => {
+			const comments:any = Object.assign({}, prev);
+			for(const key of keys ){
+				let flag = false;
+				if (prev?.[key]) {
+					comments[key] = prev[key].map((comment:any) => {
+						if (comment.id === commentId) {
+							comment.replies = comment?.replies?.filter((reply:any) => (reply.id !== replyId)) || [];
+							flag = true;
+						}
+						return {
+							...comment
+						};
+					});
+				}
+				if(flag){
+					break;
+				}
+			}
+			return comments;
+		});
+		queueNotification({
+			header: 'Success!',
+			message: `${allowed_roles?.includes('moderator')? 'The' : 'Your'} reply was deleted.`,
```

Comment:
please change message to `The reply has been deleted.`

Reply:
done, pls check